client/extensions/belgaCoverage/src/belga-coverage-preview.tsx

```diff
@@ -2,28 +2,27 @@ import * as React from 'react';
 import {IPreviewComponentProps, ISuperdesk} from 'superdesk-api';
 
 import {parseIds} from './belga-coverage-editor';
-import BelgaCoverageInfo from './belga-coverage-info';
 import BelgaCoverageImages from './belga-coverage-images';
+import BelgaCoverageCarousel from './belga-coverage-carousel';
 
 export function getBelgaCoveragePreview(superdesk: ISuperdesk): React.StatelessComponent<IPreviewComponentProps> {
     return function BelgaCoveragePreview(props: IPreviewComponentProps) {
         const ids = parseIds(props.value);
 
-        console.info('ids', ids, props.value);
-
         return (
             <React.Fragment>
                 {ids.map((id) => (
                     <div key={id}>
-                        {props.item.type !== 'graphic' &&
-                            <BelgaCoverageInfo coverageId={id} superdesk={superdesk} />
-                        }
-                        <BelgaCoverageImages
-                            coverageId={id}
-                            rendition={'preview'}
-                            maxImages={10}
-                            superdesk={superdesk}
-                        />
+                        {props.item.type !== 'graphic' ? ( // belga coverage in an article
+                            <BelgaCoverageCarousel coverageId={id} superdesk={superdesk} preview={true} />
+                        ) : ( // single belga coverage in the search, there is description already
```

Comment:
What kind of description are you referring to here?

> belga coverage in the search

do you mean preview pane opened when in search view?

To me it seems like both components render lists of images, why are different components used?

Reply:
we have search for those coverages, in that case in the preview you open just single coverage item which has name as headline and description as abstract and we only need to show images in preview on top of it, while in the other case it's embedded in an article which doesn't have the coverage name/description in the other metadata, so we display it all together